apps/api/tsconfig.json

```diff
@@ -9,5 +9,6 @@
     "allowJs": false,
     "esModuleInterop": false,
     "declarationMap": true
-  }
+  },
+  "exclude": ["node_modules"]
```

Comment:
I think this one can be reverted, since we don't really need it. I think the issues you faced previously were related to another versioning problem we had.